native/src/routes/feedback/components/FeedbackModal.js

```diff
@@ -14,43 +14,67 @@ import buildConfig from '../../../modules/app/constants/buildConfig'
 import type { SendingStatusType } from '../containers/FeedbackModalContainer'
 
 const Input = styled(TextInput)`
-  margin-bottom: 15px;
-  border-bottom-width: 1px;
-  border-bottom-color: ${props => props.theme.colors.textSecondaryColor};
+  padding: 15px;
+  border-width: 1px;
+  border-color: ${props => props.theme.colors.themeColor};
 `
 
 const Wrapper = styled.View`
   padding: 40px;
   background-color: ${props => props.theme.colors.backgroundColor};
 `
 
-const Description = styled.Text`
+const DescriptionContainer = styled.View`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
   padding: 15px 0 5px;
+`
+
+const Description = styled.Text`
+  display: flex;
+  text-align: left;
   color: ${props => props.theme.colors.textColor};
   font-family: ${props => props.theme.fonts.decorativeFontRegular};
+  font-weight: bold;
 `
 
 const RequiredText = styled.Text`
```

Comment:
I guess this is not used anymore, is it?

Reply:
it was, but i removed it now.